lib/kpeg/compiled_parser.rb

```diff
@@ -151,24 +151,26 @@ def scan(reg)
     end
 
     if "".respond_to? :ord
-      def get_byte
+      def get_byte(char_range = nil)
```

Comment:
Since we only call `get_byte` with a range one place, seems better to just have a separate method for that, perhaps `check_byte_in_range`.

Reply:
Ok, will do.
Looks like it were single place where result of get_byte were used. So `get_byte` should be also renamed.
`match_dot` ?

Reply:
Yeah, agreed. `match_dot` is best.